modules/eks_nodegroup/bootstrap/early-customizations/test_nvme.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/env bash
+
+is_nvme() {
```

Comment:
It's a minor change but I know in the shell script we have a lot about nvme. What do you think about making the function name a little bit more understandable?  for instance `is_instance_type_nvme`